pkg/ocm/billing.go

```diff
@@ -34,22 +36,63 @@ func (c *Client) GetBillingAccounts() ([]string, error) {
 		return nil, handleErr(quotaCostResponse.Error(), err)
 	}
 
-	var billingAccounts []string
+	var billingAccounts []*v1.CloudAccount
 	for _, item := range quotaCostResponse.Items().Slice() {
-		for _, account := range item.CloudAccounts() {
-			if account.CloudProviderID() == "aws" && !helper.Contains(billingAccounts, account.CloudAccountID()) {
-				billingAccounts = append(billingAccounts, account.CloudAccountID())
-			}
-		}
+		billingAccounts = append(billingAccounts, item.CloudAccounts()...)
 	}
 
 	if len(billingAccounts) == 0 {
-		return billingAccounts, errors.New("no billing accounts found")
+		return billingAccounts, errors.New("No billing account associated. " +
+			"Go to https://docs.openshift.com/rosa/rosa_architecture/rosa-understanding.html" +
+			" to learn how to associate your billing account." +
+			" You must associate a billing account to continue.")
 	}
 
 	return billingAccounts, nil
 }
 
+func GenerateBillingAccountsList(cloudAccounts []*v1.CloudAccount) []string {
+	var billingAccounts []string
+	for _, account := range cloudAccounts {
+		if account.CloudProviderID() == "aws" && !helper.ContainsPrefix(billingAccounts, account.CloudAccountID()) {
+			var contractString string
+			if HasValidContracts(account) {
+				contractString = " [Contract enabled]"
+			}
+			contractEnabledBillingAccount := account.CloudAccountID() + contractString
+			billingAccounts = append(billingAccounts, contractEnabledBillingAccount)
+		}
+	}
+	return billingAccounts
+
+}
+
+func GetNumsOfVCPUsAndClusters(dimensions []*v1.ContractDimension) (int, int) {
+	numOfVCPUs := 0
+	numOfClusters := 0
+	for _, dimension := range dimensions {
+		switch dimension.Name() {
+		case "four_vcpu_hour":
+			numOfVCPUs, _ = strconv.Atoi(dimension.Value())
+		case "control_plane":
+			numOfClusters, _ = strconv.Atoi(dimension.Value())
+		}
+	}
+	return numOfVCPUs, numOfClusters
+}
+
+func HasValidContracts(cloudAccount *v1.CloudAccount) bool {
```

Comment:
Can you leave a comment here on what is a valid contract? And a small unit test too please :pray:

Reply:
Sure, will do

Reply:
Added the comment and tests.